migration/EAP7-990_JBoss_EAP_7.2_Server_Migration_Tool.adoc

```diff
@@ -0,0 +1,147 @@
+*** infinispan
+*** jGroups
+*** undertow
+** migration of resources, which paths are referenced by migrated server configurations, such as property files, keystores and certificates
+** migration of modules referenced by the server configurations
+
+==== Migration of Modules
+
+* The migration of modules, installed in the source server, and referenced by server configurations should be included.
+* The user should be able to specify modules to be included and excluded in the migration.
+* The migration of modules should be aware that the source server may have a modules overlay installed, which may include additional and/or updated modules
+
+==== Migration of Deployments
+
+* Includes migration of managed content, regarding both deployment and deployment overlays.
+
+=== Nice-to-Have Requirements
+
+* The following server migrations may be supported:
+** JBoss EAP 7.0 to JBoss EAP 7.2
+
+* The migration of JBoss EAP 7.0 server configurations includes:
+** add default configurations for JBoss EAP 7.2 subsystems, which didn't exist in JBoss EAP 7.0, and are required for the server to work correctly
+*** core-management
+*** discovery
+*** ee-security
+*** elytron
+** subsystem configuration updates necessary for the server configurations to work as expected with JBoss EAP 7.2
+*** infinispan
+*** jGroups
+*** undertow
+** migration of resources, which paths are referenced by migrated server configurations, such as property files, keystores and certificates
+** migration of modules referenced by the server configurations
+
+=== Non-Requirements
+
+* Migration of deployments found on source server does not includes any required or optional modifications to its content.
+* Migration should not include the runtime related data present in the source server.
+
+== Implementation Plan
+
+* The migration tool for JBoss EAP 7.2 is the JBoss Server Migration Tool, which is developed as a standalone project:
+** Source repo and community binaries at https://github.com/wildfly/wildfly-server-migration[GitHub]
+** Project management at https://issues.jboss.org/projects/CMTOOL[JIRA]
+
+* The tool is not a standalone product, it’s simply a component in the JBoss EAP product, and should follow same procedures established for other JBoss EAP components with respect to Productization, QE and Support. The MIGRATION component in WFLY JIRA should be used for issues related with the integration of the tool in the JBoss EAP 7.2 distributions.
+
+* The “productized” tool should be delivered in the JBoss EAP 7.2 server distribution, by provisioning a Galleon Feature Pack specific for it, adding the following content:
+** a readme with basic info and instructions at `/migration`
+** the tool configuration files at `/migration/configuration`
+** the sh and bat scripts (jboss-server-migration.*) to start the tool at `/bin`
+** a system module with the tool’s artifacts/resources at `/modules/system/base/org/jboss/migration/cli`
+
+== Test Plan
+
+* TODO
```

Comment:
Could you please add a link to Polarion test plan here: https://polarion.engineering.redhat.com/polarion/#/project/EAP7/wiki/RFEs_7_2/EAP7-990 ?

Otherwise looking good to me.